src/viam/examples/project/pkg-config/Makefile

```diff
@@ -1,19 +1,27 @@
 CXX = c++
 
-CXXFLAGS = -std=c++14
+# Technically the SDK supports building with c++14, but note that more recent
+# versions of abseil will force c++17, so this may be needed to make things
+# work out of the box.
+CXX_STANDARD ?= c++17
```

Comment:
This seems reasonable to me but I notice we're not similarly setting the standard in the `CMakLists.txt`, is there a reason for that?

Reply:
Yeah I think the philosophy will be that in the CMake project itself we still default to c++14 but just do c++17 here to make it as get-up-and-go as possible. That said, I'm going to create a ticket to capture some of the discussions we've been having